{"name": "test_create_user[data0]", "status": "failed", "statusDetails": {"message": "AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nassert None is not None", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def init_token():\n        \"\"\"初始化环境配置\"\"\"\n        cfg = read_yaml(\"config/config.yaml\")\n>       assert cfg is not None, \"配置文件 config/config.yaml 读取失败，请检查路径或内容\"\nE       AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nE       assert None is not None\n\nconftest.py:9: AssertionError"}, "description": "创建用户", "parameters": [{"name": "data", "value": "{'name': 'test001', 'password': '123456'}"}], "start": 1761032991287, "stop": 1761032991287, "uuid": "51927ca0-473f-40da-a2a7-2da4efd704d9", "historyId": "e28480b52b9f981cba9e4efb71eca5c4", "testCaseId": "28003e8ffc50fbad52c38a02461ca6c2", "fullName": "testcases.test_user_create#test_create_user", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_create"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "17608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_create"}], "titlePath": ["testcases", "test_user_create.py"]}